tools/building.py

```diff
@@ -915,17 +915,31 @@ def move_to_safe_7bit_ascii_filename(filename):
   # temp directory.
   utils.delete_file(outfile + '.map')
 
+  closure_warnings = diagnostics.manager.warnings['closure']
+
+  if settings.CLOSURE_WARNINGS:
```

Comment:
this block looks duplicated from emcc.py, how hard would it be to avoid that?

Reply:
Oops, meant to move it.  Removing this one..